AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -51,20 +53,46 @@ struct SessionCartView: View {
                 SessionLoadingView()
             }
         }
-        .onChange(of: session.allStreams) { _ in
-            selectedStream = session.allStreams?.first
+        .onChange(of: session.sortedStreams) { newValue in
+            selectDefaultStreamIfNeeded(streams: newValue ?? [])
         }
         .onAppear {
-            selectedStream = session.allStreams?.first
+            selectDefaultStreamIfNeeded(streams: session.sortedStreams ?? [])
         }
         .font(Font.moderate(size: 13, weight: .regular))
         .foregroundColor(.aircastingGray)
         .padding()
         .background(
-            Color.white
-                .shadow(color: Color(red: 205/255, green: 209/255, blue: 214/255, opacity: 0.36), radius: 9, x: 0, y: 1)
+            Group {
+                Color.white
+                    .shadow(color: Color(red: 205/255, green: 209/255, blue: 214/255, opacity: 0.36), radius: 9, x: 0, y: 1)
+                NavigationLink(
```

Comment:
would be nice to move those navigation links to separate vars so it's easier to read thru this layout